src/constants/gender.js

```diff
@@ -0,0 +1,2 @@
+export const MALE_GENDER = 'male';
```

Comment:
Is it intentionally? There's the same file with plural name

https://github.com/BuildIt-Poland/daily-routine/blob/master/src/constants/genders.js